SCons/Environment.xml

```diff
@@ -78,56 +78,60 @@ env['BUILDERS']['NewBuilder'] = bld
 <cvar name="ENV">
 <summary>
 <para>
-A dictionary of environment variables
-to use when invoking commands. When
-&cv-ENV; is used in a command all list
-values will be joined using the path separator and any other non-string
-values will simply be coerced to a string.
-Note that, by default,
-&scons;
+The <firstterm>execution environment</firstterm> -
```

Comment:
do we want to be more explicit here and say is the shell environment provided to the SPAWN specified command?

Reply:
Punt this and I'll merge.